.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,52 +1,69 @@
 name: General Bug Report
-description: Create a report to help us improve
+description: Report a bug or other issue to help us improve our products.
```

Comment:
our products => the Anaconda Installers, Anaconda Navigator, Anaconda built packages and Anaconda.org.

Reply:
Thanks @csoja. Navigator has [their own form](https://github.com/ContinuumIO/anaconda-issues/issues/new?assignees=&labels=type-bug%2Cpkg-navigator&template=navigator_bug_report.yml), but I added the others.